### PR TITLE
Fix previous / next links in bsrequest show view

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/app/controllers/webui/request_controller.rb [hunk 1]
```diff
@@ -244,10 +241,9 @@ def diff
   def list
     redirect_to user_show_path(User.current) and return unless request.xhr? # non ajax request
     requests = BsRequest.list_ids(params)
-    elide_len = 44
-    elide_len = params[:elide_len].to_i if params[:elide_len]
-    session[:requests] = requests
-    requests = BsRequest.collection(ids: session[:requests])
+    elide_len = (params[:elide_len] || 44).to_i
+    session[:request_numbers] = requests.map { |id| BsRequest.find(id).number }.uniq
+    requests = BsRequest.collection(ids: requests)
```
Comment (bgeuken) [inline src/api/app/controllers/webui/request_controller.rb]: I almost think we could replace using ids by using numbers here. But I won't be around next week. So no time to dive deeper.

**Comment by adrianschroeter** [comment]
can you please also add some test cases documenting what we have not tested so far? having them just as an empty skip test case is okay, but that we do not loose the information what is missing ....

**Comment by bgeuken** [comment]
I've added a test now

**Comment by mdeniz** [comment]
L
